torchgeo/transforms/indices.py

```diff
@@ -201,3 +201,39 @@ def __init__(self, index_nir: int, index_green: int) -> None:
             index_green: index of the Green band, e.g. B3 in Sentinel 2 imagery
         """
         super().__init__(index_a=index_nir, index_b=index_green)
+
+
+class AppendBNDVI(AppendNormalizedDifferenceIndex):
+    """Blue Normalized Difference Vegetation Index (BNDVI).
+
+    If you use this index in your research, please cite the following paper:
+
+    * https://doi.org/10.1016/S1672-6308(07)60027-4
```

Comment:
Can you add:
```rst
.. versionadded:: 0.3
```
to these new indices you've added?

Reply:
i have 2 questions 
1.Where shall i put the formulae in latex ?
2.Where shall i have to add .. versionadded:: 0.3


> Can you update the description and title to explain what indices are being added?

Reply:
1. It's probably better to do this in a follow-up PR since we also need to do it for existing indices, let's discuss this more in #387 
2. At the bottom of the class docstring. If you grep the rest of TorchGeo for `versionadded` you'll see other examples of this

Reply:
Added the version 0.3.0 to the added indices.